app/src/main/java/com/dimelo/sampleapp/RcConfig.java

```diff
@@ -18,7 +22,12 @@ public static Dimelo setupDimelo(Context context) {
         dimelo.initWithApiSecret(secret, domainName, null);
         dimelo.setDebug(true);
         dimelo.setUserName("John Doe");
-        dimelo.setThreadsEnabled(true);
+        boolean isThreadEnabled = RcConfig.getBooleanValueFromSharedPreference(context,RC_THREAD_ENABLED);
+        setThreadsEnabled( context, RC_THREAD_ENABLED, isThreadEnabled);
+        String userIdVal = RcConfig.getStringValueFromSharedPreference(context,RC_USER_ID);
```

Comment:
```suggestion
        String userIdVal = RcConfig.getStringValueFromSharedPreference(context, RC_USER_ID);
```

And everywhere else, I won't comment them all.

Reply:
To be sure, did you see that I added this sentence too?

> And everywhere else, I won't comment them all.

Reply:
yes

Reply:
You mean you didn't push it yet?

Reply:
yes !

Reply:
You didn't push it yet? I still see this in the diff:

```
         SharedPreferences sharedPref = context.getSharedPreferences("RCSHAREDPREF",Context.MODE_PRIVATE);
```

So I'm asking again, does our linter work in the sample app? Could it detect these cases?

Reply:
no not working for the sample app